### ISSUE TITLE
VTK documentation

### ISSUE THREAD

**Issue by jlchan** [jlchan/StartUpDG.jl#85]
@Davknapp when the VTK functionality is mature, we should add some documentation describing how to use it.

**Comment by Davknapp** [comment]
Yes, I agree. 
What exactly do you mean by mature? In my opinion we need, a VTK-Writer for all 3D elements to call it mature. A nice addition would then be to have a look at hybrid meshes but in my opinion not necessary for a mature version of the VTK-Writer

**Comment by jlchan** [comment]
I just meant when the interface is settled and not likely to change. I think we could add it after adding 3D mesh capabilities, but I don't think we need to cover every element before that (since you're probably focusing more on prisms at first).

**Comment by Davknapp** [comment]
Yes, the interface probably won't change after adding the 3D writer. I don't know if it will change again when we use it for hybrid meshes in the feature.

**Comment by jlchan** [comment]
I think this is addressed enough in https://github.com/jlchan/StartUpDG.jl/pull/194 to close this issue.
